README/ReleaseNotes/v624/index.md

```diff
@@ -98,9 +98,42 @@ See the discussion at [ROOT-11014](https://sft.its.cern.ch/jira/browse/ROOT-1101
 
 ## RooFit Libraries
 
+- Extension / updates of the doxygen reference guide.
+- Allow for removing RooPlot from global directory management, see [RooPlot::AddDirectory](https://root.cern/doc/v624/classRooPlot.html#a47f7ba71dcaca30ad9ee295dee89c9b8)
+  and [RooPlot::SetDirectory](https://root.cern/doc/v624/classRooPlot.html#a5938bc6d5c47d94c2f04fdcc10c1c026)
+- Hash-assisted finding of elements in RooWorkspace. Large RooWorkspace objects were slow in finding elements.
+  This was improved using a hash map.
+- Stabilise RooStats::HypoTestInverter. It can now tolerate a few failed fits when conducting hypothesis tests.
+  This is relevant when a few points in a parameter scan don't converge due to numerical or model instabilities.
+  These points will be skipped, and HypoTestInverter can continue.
+- New PDFs
```

Comment:
The bullet point `New PDFs` is not up to date anymore since `RooDSCBShape` and `RooSDSCBShape` were merged into `RooCrystalBall`. There is already a [release notes](https://github.com/root-project/root/blob/master/README/ReleaseNotes/v624/index.md#new-fully-parametrised-crystal-ball-shape-class) section about this now.

Sorry for the inconvenience! I should have noticed it in https://github.com/root-project/root/pull/7452 already, but I learned about this PR only when it was too late...

Reply:
I know, I'm on it. I'll also fix the stray bullet point in your DataHist::sum() while I'm at it. 👍

Reply:
Thank you very much!